app/jobs/regular/export_csv_file.rb

```diff
@@ -198,8 +198,25 @@ def report_export
         end
       end
 
+      if report.modes == [:stacked_chart]
```

Comment:
When we're exporting a `stacked_chart` report, we should probably avoid doing the loop above (starting on line `191`)

Reply:
That loop is necessary to populate the `titles` hash. 😊